nightingale_interface/screens/admin_screen.py

```diff
@@ -0,0 +1,91 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivymd.uix.button import MDRectangleFlatButton
+from kivy.uix.button import Button
+
+from kivy.uix.screenmanager import SlideTransition, NoTransition
+
+from screens.screen_config import ScreenConfig as cfg
+
+
+class AdminScreen:
```

Comment:
access to the admin screen should be password protected. Not something we need in this PR just commenting it to keep it in mind for later

Reply:
yeah I have that written down but it's something that I didn't think was a priority for now. However I think I might have time this weekend so I can probably do it then in another PR